package.json

```diff
@@ -4,26 +4,34 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "cross-env NODE_ENV=development nyc mocha --require @babel/register --require @babel/polyfill -r dotenv/config ./src/__tests__/*.js --exit",
+     "test": "cross-env NODE_ENV=test nyc mocha --require @babel/register --require @babel/polyfill -r dotenv/config ./src/__tests__/*.js --exit",
     "coverage": "nyc report --reporter=text",
     "coveralls": "nyc report --reporter=text-lcov | coveralls",
     "lint": "eslint \"**/*.js\" --ignore-pattern node_modules/ --fix",
     "build": "babel ./src -d dist",
     "start": "node dist/index.js",
-    "start:dev": "nodemon --exec babel-node -- src/index.js",
-    "migrate:dev": "sequelize db:migrate",
+    "start:dev": "cross-env NODE_ENV=development nodemon --exec babel-node -- src/index.js",
+    "migrate:dev": "cross-env NODE_ENV=development sequelize db:migrate",
```

Comment:
There is no need of specifying `cross-env NODE_ENV=development` since we are only executing this in development mode

Reply:
ok, thanks